checkov/runner_filter.py

```diff
@@ -262,6 +264,10 @@ def within_threshold(self, severity: Severity) -> bool:
         below_max = self.skip_check_threshold and self.skip_check_threshold.level >= severity.level
         return above_min and not below_max
 
+    @staticmethod
+    def secret_check_matches(secret_validation_status: str, statuses_list: list[str]) -> bool:
```

Comment:
Is check_matches supposed to run only after the status of the secret is validated? 
If so - maybe the name can reference this, as `post_secret_check_matches ` or something like this.
I initially thought this is a similar method to `check_matches` but for `secrets`

Reply:
changed to `secret_validation_status_matches`, wdyt?